caffe2/operators/gather_fused_8bit_rowwise_op.cc

```diff
@@ -0,0 +1,40 @@
+#include "caffe2/operators/gather_fused_8bit_rowwise_op.h"
+
+namespace caffe2 {
+
+OPERATOR_SCHEMA(GatherFused8BitRowwise)
+    .NumInputs(2)
+    .NumOutputs(1)
+    .SetDoc(R"DOC(
+Perform the same operation as Gather, but operating on 8-bit rowwise quantized
+matrices with fused storage (where each row stores quantized values, and then
+the scale and offset).
+)DOC")
```

Comment:
Unlike Gather, which operates on DATA and INDICES tensors of arbitrary rank, this requires rank(DATA) == 2 and rank(INDICES) == 1. Probably worth documenting that difference/limitation here since it is enforced in the implementation?